Sources/Document/Document.swift

```diff
@@ -170,10 +212,28 @@ class Document {
     }
 
     /**
-     * `toJSON` returns the sorted JSON encoding of this array.
+     * `toSortedJSON` returns the sorted JSON encoding of this array.
+     */
+    private func toSortedJSON() -> String {
+        return self.root.debugDescription
+    }
+
+    /**
+     * `applySnapshot` applies the given snapshot into this document.
      */
-    func toSortedJSON() -> String {
-        return self.root.toSortedJSON()
+    func applySnapshot(serverSeq: Int64, snapshot: Data?) {
+        // TODOs Converter.bytesToObject is not implemented yet.
```

Comment:
I made bytesToObject() function in another repo. I'll change here, after the merge.